src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,87 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as TodoAction } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+  const dispatch = useAppDispatch();
+
+  const handleModal = async () => {
+    if (!currentTodo) {
+      return;
+    }
+
+    setIsLoading(true);
+
+    try {
+      const getUserFromServer = await getUser(currentTodo.userId);
```

Comment:
What if this request will fail?
![Screenshot 2023-06-12 at 11 34 33](https://github.com/mate-academy/react_redux-list-of-todos/assets/54940595/ff2a9d7f-ad8f-4d3f-b6fc-769b763746d8)